vss_invoice_mgmt/src/components/Vendor-State/Vendor_Statedropdown.jsx

```diff
@@ -0,0 +1,43 @@
+import * as React from "react";
```

Comment:
State dropdown would be generalize not specific for vendor. Change name to "State-Dropdown"

Reply:
Changed